src/clj/game/core/costs.clj

```diff
@@ -19,19 +19,19 @@
 (defn toast-msg-helper
   "Creates a toast message for given cost and title if applicable"
   [state side cost]
-  (let [type (first cost)
+  (let [c-type (first cost)
```

Comment:
I'd call this `cost-type`

Reply:
Also, I wonder if we can use expansion here instead of `first` and `last`. So instead `[[cost-type amount] cost]`

Reply:
Note for others who read this and can't find anything about "expansion": it's called destructuring, and it's hella cool! I found the docs here: https://clojure.org/guides/destructuring